src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraMethodsSpec.scala

```diff
@@ -107,11 +105,16 @@ class AgoraMethodsSpec extends ApiServiceSpec {
   }
 
   "Agora" should "store 10kb of github markdown as method documentation and return it without alteration" in {
-    Post(ApiUtil.Methods.withLeadingSlash, testAgoraEntityBigDoc) ~>
+    Post(ApiUtil.Methods.withLeadingSlash + "?includedField=documentation", testAgoraEntityBigDoc) ~>
       methodsService.postRoute ~> check {
       handleError(
         entity.as[AgoraEntity],
-        (entity: AgoraEntity) => assert(entity.documentation.get === bigDocumentation.get)
+        (entity: AgoraEntity) => {
+          val searchEntity = new AgoraEntity(name = entity.name, namespace = entity.namespace, snapshotId = entity.snapshotId)
```

Comment:
Why do we need to perform a separate search? Shouldn't the return packet of the method post desearialize to an entity that includes the documentation?

I'm a little unclear what's going on with the changes in this test